torch/csrc/jit/dynamic_dag.h

```diff
@@ -447,41 +479,43 @@ bool DynamicDAG<T>::dfsSearch(
 
 // Reorder deltaB vertices to occur before deltaF vertices.
 template <typename T>
-void DynamicDAG<T>::reorder(vertex_list<T>& deltaF, vertex_list<T>& deltaB) {
-  sort(deltaB);
-  sort(deltaF);
+void DynamicDAG<T>::reorder(visited_list<T> deltaF, visited_list<T> deltaB) {
+  auto deltaB_ = deltaB.vector();
```

Comment:
can you move those vectors? It will be a copy otherwise.

Reply:
Do you mean rewrite this as `reorder(visited_list<T>&& deltaF, visited_list<T>&& deltaB)`? (Sorry I am not very good with rvalues).

Right now I am assuming that the compiler will move those values for me (I'm calling `reorder(std::move(deltaF), std::move(deltaB)`) but that is probably not correct...

Reply:
oh of course! thank you for the explanation. Making deltaB.vector() return a `const std::vector<Vertex<T>*>&` sounds like the way to go